completions/ssh.completion.sh

```diff
@@ -45,7 +45,7 @@ function _omb_completion_ssh {
     fi
   done
   if ((${#config_files[@]} != 0)); then
-    COMPREPLY+=($(compgen -W "$(awk '/^[Hh][Oo][Ss][Tt] /{for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
+    COMPREPLY+=( $(compgen -W "$(awk 'BEGIN {IGNORECASE=1}/^[Hh][Oo][Ss][Tt] / {current_host=""; for (i=2; i<=NF; i++) hosts[current_host=$i]=""}/^[Hh][Oo][Ss][Tt][Nn][Aa][Mm][Ee] / && current_host {hosts[current_host]=$2} END {for (host in hosts) print hosts[host] ? hosts[host] : host}' "${config_files[@]}")" "${options[@]}") )
```

Comment:
- `IGNORECASE` is the GNU extension, while Oh My Bash doesn't only target GNU systems. Actually, since you've specified the patterns as `[Hh][Oo]...`, I think you don't need to set that variable.
- As far as I understand the updated version, the name specified by `Host` seems to be disabled if `HostName` is defined. However, suggesting both `Host` and `HostName` is better. If only either one is shown, it should suggest `Host` (but not `HostName`). Could you fix this?